lib/tasks/route_import.rake

```diff
@@ -6,31 +6,10 @@ require 'csv'
 
 namespace :routes do
   task :import, [:csv_file] => :environment do |_, args|
-    stop_hash = {}
-    CSV.foreach(args[:csv_file], headers: true, col_sep: ';') do |row|
-      route = row['rte_identifier']
-      variant = row['variant']
-      dir = row['direction']
-      stop = row['stp_identifier']
-      sequence = row['stop_variant_rank'].to_i
-
-      stop_hash[[route, dir]] ||= {}
-      stop_hash[[route, dir]][variant] ||= []
-      stop_hash[[route, dir]][variant][sequence] = stop
-    end
-    BusStopsRoute.import(stop_hash)
-    
     Route.delete_all
-    BusStopsRoute.delete_all
-    
-    stop_hash.each_pair do |(route, direction), stop_ids|
-      Route.find_or_create_by! number: route.strip
-      stop_ids.each.with_index(1) do |stop_id, sequence|
-        stop = BusStop.find_by(hastus_id: stop_id)
-        if stop.present?
-          BusStopsRoute.create!(route: route, direction: direction, bus_stop: stop, sequence: sequence)
-        end
-      end
+    CSV.foreach(args[:csv_file], headers: true, col_sep: ';') do |row|
+      route_number = row['route_id']
+      Route.find_or_create_by! number: route_number.strip
```

Comment:
Since we're not using my latest OIG to import BSRs, we can use it to just get a flat listing of routes, and then we can get description back. I don't think it's out of scope to roll that into a PR where we use GTFS to import BSRs, but open to being overruled on that.

Reply:
You've used one too many acronyms, my friend. I'd be happy to do whatever it is. I have two questions:
1. What is OIG?
2. By description, do you mean "direction"? If so, I see that yeah, there's direction there, but what kind of gymnastics do I have to do to figure out which direction from this file goes with which longest_trip on that other one? I can't fathom how to match them easily. If you can please share. Or go for it.

Reply:
This is more an item for @mboneil10. Can table for this PR and do another subsequent PR.

Reply:
Sounds good.